AngularApp/src/app/modules/main/main/main.component.ts

```diff
@@ -71,6 +73,10 @@ export class MainComponent implements OnInit {
     this._http.get('assets/enabledResourceTypes.json').map(response => {
       this.enabledResourceTypes = <ResourceServiceInputs[]>response.json().enabledResourceTypes;
     });
+
+    if (_adalService.userInfo.userName === 'cmaher@microsoft.com' || _adalService.userInfo.userName === "shgup@microsoft.com"){
```

Comment:
This value was not populated during my testing, did it work for you? I was using the following `this._adalService.userInfo.profile.upn`

Reply:
It works for my user name at least, although I would like this to be a DL or security group but I don't know how to do that